middleware/archives_queries.py

```diff
@@ -54,7 +54,7 @@ def update_archives_data(
     """
     if broken_as_of:
         db_client.update_url_status_to_broken(data_id, broken_as_of, last_cached)
-    else:
-        db_client.update_last_cached(data_id, last_cached)
+    
```

Comment:
I'm interested in the removal of this else conditional -- if I recall correctly, a broken url should not be archived, correct? I may be wrong on that (and am happy to be corrected if that's the case). That being said, such a source could still count as being "archived", but it's a tad ambiguous. Could you clarify your reasoning with this decision?

Reply:
Okay this is going to be a little complicated to explain so let me know if I'm explaining it incoherently:
If you look at what was previously line 451 of `database_client.py` in `update_url_status_to_broken()` I had to remove `"last_cached": last_cached,` as part of the data package being sent to `update_data_source()` since `last_cached` is no longer part of the `data_sources` table. Since `last_cached` was updated in both the `if` and `else` branches, removing the `else` means that `last_cached` will be updated regardless of if it is broken or not, just like previously.

Reply:
But I was also a little confused why it was set up this way in the first place. Why, when a link is broken do we update the `last_cached` column? Surely the source has not been archived if it is broken. However, I didn't want to change the logic unless you believe it should be now or in the future

Reply:
Well, I don't know the original design decision in this case, but in my thinking there is a possibility that a link was _previously_ operational and then archived, so last_cached might still be applicable in that context. But it is also a case where I think we could clarify the design for this later on.

Reply:
At any rate, you've convinced me with this! _**RESOLVED**_.